test/src/module/command/backupTest.c

```diff
@@ -1396,7 +1396,7 @@ testRun(void)
         // -------------------------------------------------------------------------------------------------------------------------
         TEST_TITLE("report job error");
 
-        ProtocolParallelJob *job = protocolParallelJobNew(VARSTRDEF("key"), protocolCommandNew(STRDEF("command")));
+        ProtocolParallelJob *job = protocolParallelJobNew(VARSTRDEF("key"), protocolCommandNew(strIdFromZ(stringIdBit6, "X")));
```

Comment:
It doesn't actually matter what you set here, e.g. `strIdFromZ(stringIdBit5, "bp-f")` instead of `strIdFromZ(stringIdBit6, "X")` would still fail because the error is set via the next line `protocolParallelJobErrorSet`, right?

Reply:
You are right, it does not matter. But let's just make these 5-bit to avoid confusion. Done in d8902130e0.